apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -182,37 +224,94 @@ export class BulkScriptAction extends BaseScriptAction {
       parameterLongName: '--to',
       parameterShortName: '-t',
       argumentName: 'PROJECT',
       description:
-        'Run command in the specified project and all of its dependencies. "." can be used as shorthand ' +
-        'to specify the project in the current working directory.',
+        'Run command on the selection instead of all projects. ' +
+        'Adds the specified project and all its dependencies to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of any selection commands will further expand the selection.',
       completions: this._getProjectNames.bind(this)
     });
-    this._fromVersionPolicy = this.defineStringListParameter({
-      parameterLongName: '--from-version-policy',
-      argumentName: 'VERSION_POLICY_NAME',
+    this._toExceptProject = this.defineStringListParameter({
+      parameterLongName: '--to-except',
+      parameterShortName: '-T',
+      argumentName: 'PROJECT',
       description:
-        'Run command in all projects with the specified version policy ' +
-        'and all projects that directly or indirectly depend on projects with the specified version policy'
+        'Run command on the selection instead of all projects. ' +
+        'Adds all dependencies of the specified project to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of any selection commands will further expand the selection.',
+      completions: this._getProjectNames.bind(this)
     });
+
+    this._fromProject = this.defineStringListParameter({
+      parameterLongName: '--from',
+      parameterShortName: '-f',
+      argumentName: 'PROJECT',
+      description:
+        'Run command on the selection instead of all projects. ' +
+        'Add the specified project and all projects that depend on it, and all the dependencies of those projects, to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of any selection commands will further expand the selection.',
+      completions: this._getProjectNames.bind(this)
+    });
+    this._onlyProject = this.defineStringListParameter({
+      parameterLongName: '--only',
+      parameterShortName: '-o',
+      argumentName: 'PROJECT',
+      description:
+        'Run command on the selection instead of all projects. ' +
+        'Add the specified project (and only the specified project) to the current selection. ' +
+        '"." can be used as shorthand to specify the project in the current working directory. ' +
+        'Additional use of any selection commands will further expand the selection.',
+      completions: this._getProjectNames.bind(this)
+    });
+
+    this._affectedByProject = this.defineStringListParameter({
+      parameterLongName: '--affected-by',
```

Comment:
I think our final spec called it `--impacted-by` with `-i`.

Here's the comment from Teams for reference:

```
rush build --to project
rush build -t project

rush build --to-except project
rush build -T project

rush build --from project
rush build -f project

rush build --only project
rush build -o project

rush build --impacted-by project
rush build -i project

rush build --impacted-by-except project
rush build -I project

rush build --changed-projects-only
rush build -c
```

Reply:
Updated.